src/modules/recipes/RecipeList.js

```diff
@@ -13,23 +17,52 @@ export const RecipeList = ({ recipes, onRecipeClick }) => {
     }
   };
 
-  return (
-    <div className="recipe-list">
-      {Array.isArray(recipes) && recipes.length > 0 ? (
-        <ul>
-          {recipes.map((recipe) => (
-            <li
-              className="recipe-item"
-              key={recipe.idMeal}
+  return Array.isArray(recipes) && recipes.length > 0 ? (
+    <Box
+      sx={{
+        position: "absolute",
```

Comment:
I would not recommend using absolute position to place this element on the page. Instead, make the parent element a flex container, then center this element inside of it using flexbox.

Also, it would provide better user experience if this box's height would not be restricted. So allow it to e as height as page and scroll naturally.